src/attr/_make.py

```diff
@@ -2563,7 +2660,9 @@ def _setattrs(self, name_values_pairs):
         repr=True,
         cmp=None,
         eq=True,
+        # eq_key=None,
```

Comment:
?

Reply:
Apologies. I have uncommented those lines to specify a value (instead of relying on the default).